src/Core/Options.idr

```diff
@@ -220,7 +221,7 @@ defaultSession : Session
 defaultSession = MkSessionOpts False CoveringOnly False False Chez [] 1000 False False
                                defaultLogLevel Nothing False Nothing Nothing
                                Nothing Nothing False 1 False True
-                               False [] False False
+                               False False [] False False
```

Comment:
I wonder if the default should be `True`?

Reply:
The rationale: if you don't know that what's happening is a fixity conflict, it can be quite frustrating to diagnose.
If you get the warning, it's straightforward to suppress with the option.

Reply:
Ah, indeed, that was my original intent, however this creates some issues with bootstrapping and some tests relying on -Werror

Reply:
Can we easily fix the warnings that make the errors break?
(e.g. by passing a `--no-prelude`?)

Reply:
I don't think we can during bootstrap because the old version does not parse `%hide Namespace.infix.(op)`

Reply:
looks like bootstraping is fine actually, only frex stops CI now, I'll remove the compiler option since there is no need to let people rely on buggy fixities